Tests/NimbleTests/AsyncTimerSequenceTest.swift

```diff
@@ -0,0 +1,18 @@
+import XCTest
+import Foundation
+@testable import Nimble
+
+final class AsyncTimerSequenceTest: XCTestCase {
+    func testOutputsVoidAtSpecifiedIntervals() async throws {
```

Comment:
I finally realized why these tests are failing in CI. They're an example of a [falsehood programmers believe about time](https://infiniteundo.com/post/25326999628/falsehoods-programmers-believe-about-time). Namely, because CI will run these tests in a VM, the timer might actually output results differently than the 10 millisecond interval. So, we can't rely on these actually outputting dates we expect. Running on real hardware, these tests are much more accurate.

I'll have to re-architect this to be much more tolerant of that fact.